packages/xo-server-backup-reports/src/index.js

```diff
@@ -145,73 +220,176 @@ class BackupReportsXoPlugin {
     this._xo.on('job:terminated', this._report)
   }
 
-  test({ runId }) {
-    return this._backupNgListener(undefined, undefined, undefined, runId)
+  async test({ runId }) {
+    return this._report(runId)
   }
 
   unload() {
     this._xo.removeListener('job:terminated', this._report)
   }
 
-  _wrapper(status, job, schedule, runJobId) {
-    if (job.type === 'metadataBackup') {
-      return
-    }
+  async _report(runJobId, { type, status, force } = {}) {
```

Comment:
Where is `force: true` passed?

Reply:
I think `force` is internal to this plugin and not related to the `job:terminated` event, thus it should probably be separated.